scripts/reserved-names/transfer-names.ts

```diff
@@ -44,7 +45,7 @@ const parseOwnedObjects = () => {
 
 // Parses the combined CSV
 const parseCsvFile = () => {
-	fs.readFileSync('./reserved-names/data/transfers-v4.csv')
+	fs.readFileSync('./reserved-names/sample/sample.csv')
```

Comment:
nit: 
would follow previous filename format and create `data/transfers-v5.csv` instead of a sample

Reply:
oh, thanks for the clarity! 🙈